arches/app/models/graph.py

```diff
@@ -563,19 +564,18 @@ def delete(self):
                 )
             )
 
-    def delete_instances(self, verbose=False):
+    def delete_instances(self, userid=None, verbose=False):
         """
         deletes all associated resource instances
 
         """
-        if verbose is True:
-            bar = pyprind.ProgBar(Resource.objects.filter(graph_id=self.graphid).count())
-        for resource in Resource.objects.filter(graph_id=self.graphid):
-            resource.delete()
-            if verbose is True:
-                bar.update()
-        if verbose is True:
-            print(bar)
+
+        bulk_deleter = BulkDataDeletion()
+        loadid = uuid.uuid4()
+        resp = bulk_deleter.delete_resources(userid, loadid, graphid=self.graphid, verbose=verbose)
+        bulk_deleter.index_resource_deletion(loadid)
```

Comment:
This will drive more traffic into #10183, but that's not something that needs to block this work. Just FYI.